axolotls/demo/criteo.py

```diff
@@ -32,3 +37,8 @@
 df["dense1"] = (df["dense1"] + 3).log()
 df["dense2"] = (df["dense2"] + 3).log()
 print(f"df\n{df}\n")
+
+df["dense_grp"].fill_null_(0)
```

Comment:
cc @dracifer . After iteration I feel nested struct may fit better for dense feature group? (Starting with `List/FixedSizeList` as in https://github.com/wenleix/axolotls/pull/6/files#diff-23eb52e34a4462ccfe4c11b0aacf597da31a889993598d52cafae1c54928e5f7R29 . But the layout is not optimal? 

I guess similar to `List vs. FixedSizeList`, there might be a need for `Struct vs. FixedTypeStruct` so it can allow all filed columns to be a contiguous memory; but then lose the capability of replacing individual child field? Guess still need to iterate a bit...

Reply:
I am actually thinking that we just store the 2D tensors as the transpose of the current row oriented layout.